src/components/transport_manager/src/usb/libusb/usb_connection.cc

```diff
@@ -159,26 +159,28 @@ void UsbConnection::OnInTransfer(libusb_transfer* transfer) {
   LOG4CXX_TRACE(logger_, "exit");
 }
 
-void UsbConnection::PopOutMessage() {
+TransportAdapter::Error UsbConnection::PopOutMessage() {
   LOG4CXX_TRACE(logger_, "enter");
   bytes_sent_ = 0;
+  auto error_code = TransportAdapter::OK;
   if (out_messages_.empty()) {
     current_out_message_.reset();
   } else {
     current_out_message_ = out_messages_.front();
     out_messages_.pop_front();
-    PostOutTransfer();
+    error_code = PostOutTransfer();
   }
   LOG4CXX_TRACE(logger_, "exit");
+  return error_code;
 }
 
-bool UsbConnection::PostOutTransfer() {
+TransportAdapter::Error UsbConnection::PostOutTransfer() {
   LOG4CXX_TRACE(logger_, "enter");
   out_transfer_ = libusb_alloc_transfer(0);
   if (0 == out_transfer_) {
```

Comment:
@SNiukalov compare it with `nullptr` please

Reply:
@AByzhynar 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3332/commits/581a1b9d0e927f22754b6baa59c6df89ec19c721